dev/dags/basic_cosmos_dag.py

```diff
@@ -18,7 +18,7 @@
     profile_mapping=PostgresUserPasswordProfileMapping(
         conn_id="airflow_db",
         profile_args={"schema": "public"},
-        disable_event_tracking=True,
+        dbt_config_vars=DbtConfigVars(send_anonymous_usage_stats=True),
```

Comment:
Should keep `disable_event_tracking` here.

Reply:
fix